### DISCUSSION TITLE
[8.0 QA] MDX

### DISCUSSION THREAD

**Discussion by vanessayuenn** [storybookjs/storybook#25689] [Announcement]
#### Feature owner: @yannbf 
Please comment in this discussion thread for any bug report and feedback for this feature as you test out Storybook 8. Kindly adhere to the following format:

> **Project**: Name of your project as entered in Notion
> **Version**: The version of storybook you tested with
> **Bug/feedback**: Details of the bugs/feedback you have

Feature owner should triage the comments here frequently and open issues as needed.

**Comment by yannbf** [comment]
**Project:** MealDrop
**Version:** 8.0.0-alpha.14
**Bug/feedback:** 

The source snippets are not highlighted in mdx docs anymore:

<img src="https://github.com/storybookjs/storybook/assets/1671563/327453d9-9e6f-4248-bafc-ae238b6dd704" width="400" />

In comparison to before:
<img src="https://github.com/storybookjs/storybook/assets/1671563/dde9b983-52d5-4803-b4b7-943cd720476c" width="400" />

Details and reproduction are provided in this issue:
https://github.com/storybookjs/storybook/issues/25737
